_i18n/en.yml

```diff
@@ -763,3 +763,8 @@ blog:
 
 tags:
   notags: There are no posts for this tag.
+
+accessibility:
+  arrowup: "'Back to top' button"
+  guiscreen: GUI wallet screenshot
```

Comment:
Maybe this could have a bit more detail, something like:

>A screenshot of the Monero GUI wallet. It shows the wallet's balance and a navigation menu on the left, and a form for sending XMR on the right.

It's the best thing to do for images, but I'm not sure how useful it will be when showcasing GUI software (do screenreaders work with the GUI wallet?) and it's another string that will need translating.

Reply:
Yeah that would be better. I added it last minute, i agree making it mroe verbose makes sense. No problem with adding another string for translators. We can also add explanations of the context is tha string is not self explanatory.

> do screenreaders work with the GUI wallet?

I don't really know.

Reply:
Added both your suggestions.